.travis.yml

```diff
@@ -3,16 +3,16 @@ sudo: false
 cache:
   directories:
     - $HOME/.cache/pip
-python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
-  - "3.7-dev"
-  - "pypy-5.6.0"
 matrix:
-  allow_failures:
-    - python: "3.7-dev"
+  include:
+    - python: '2.7'
```

Comment:
There's no need to move it out of matrix generation, list only 3.7 in `matrix.include`, because you need to customize it.